subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -1808,22 +1816,7 @@ static inline int rx_demux_rx(memq_link_t *link, struct node_rx_hdr *rx)
 			break;
 		}
 
-		switch (rx->type) {
-		case NODE_RX_TYPE_EXT_1M_REPORT:
-			phy = BIT(0);
-			break;
-		case NODE_RX_TYPE_EXT_2M_REPORT:
-			phy = BIT(1);
```

Comment:
The assignment of `phy = BIT(0)` and `phy = BIT(2)` have been moved, but I don't see anything that sets `phy = BIT(1)` anymore; is that intentional?